docs/source/Installation.rst

```diff
@@ -0,0 +1,57 @@
+Installation
+============
+
+SyReC is mainly developed as a C++ library.
+In order to make the tool as accessible as possible, it comes with an easy-to-use Python interface.
+
+We encourage installing SyReC via pip (preferably in a `virtual environment <https://docs.python.org/3/library/venv.html>`_):
+
+    .. code-block:: console
+
+        (venv) $ pip install mqt.syrec
+
+Once installed, start the SyReC editor GUI by running:
+
+    .. code-block:: console
+
+        (venv) $ syrec-editor
+
+In most practical cases (under 64-bit Linux, MacOS incl. Apple Silicon), this requires no compilation and merely downloads and installs a platform-specific pre-built wheel.
```

Comment:
This list is missing windows. (Another copy-paste mistake). See QCEC

Reply:
done 👍🏻